SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -394,6 +408,23 @@ private void StartRandomTriggerCallbacksAsync()
             }).Start();
         }
 
+        private Task runScript(string callbackId)
```

Comment:
Follow casing convention.

Also, this method is easily confused with Run. What's the difference? Better name?